### ISSUE TITLE
Tensor Parallel Inside notebook

### ISSUE THREAD

**Issue by nivibilla** [pytorch-labs/gpt-fast#167]
Hi,

Im trying to get an example working with Ray on Databricks. Essentially having multiple replicas of the model. Is it possible to load a model with tensor parallelism inside a notebook?

Thanks

**Comment by yanboliang** [comment]
Yes, you can. gpt-fast uses ```torchrun``` to automatically dispatch tasks to different GPU nodes. You can use it just like regular python script.

**Comment by kwen2501** [comment]
Is notebook compatible with torchrun? Last time I heard it has limited (or none) multi-processing support.

**Comment by yanboliang** [comment]
Right, I think you have to find a notebook which support multiprocessing.
